trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name
+  end
+end
```

Comment:
The trimmer decorate should make sure that the the name has a maximum of 10 characters. If it's longer it should trim the word.

You have submitted a `trimmer_decorator` file that is a duplicate of the `CapitalizeDecorator`. As a result, when testing the features the trimmer decorator doesn't exit.

![image](https://github.com/GhostEsso/School_library/assets/38088818/cc7bc87f-218f-4a85-9c3f-acf10d53dffe)

- Kindly make sure that the class is named `TrimmerDecorator` or any relevant class name.
- Please make sure that a condition that trims the name if it is longer than 10 characters is implemented.

suggestion:

```ruby
...
    def correct_name
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
    end
...
```